rules.yaml

```diff
@@ -572,7 +572,10 @@
 # Ruby
 - { namepat: "ruby-(.*)",              setname: "ruby:$1", family: [ debian, arch, fedora, mageia, sisyphus, guix ] }
 - { namepat: "rubygem-(.*)",           setname: "ruby:$1", family: [ freebsd, fedora, opensuse, slackbuilds ] }
+- { namepat: "rubygem-(.*)[0-9]",      setname: "ruby:$1", family: [ freebsd ] }
```

Comment:
These two rules do nothing, since names are already processed by `rubygem-(.*)` rule. Let's not treat gems ending with number specially for now.